database.env

```diff
@@ -0,0 +1,2 @@
+// production database
```

Comment:
Do we need this file? We aren't making any API calls to the backend at this point